update-images.sh

```diff
@@ -20,11 +19,10 @@ while IFS= read -r VERSION
 do
 
   # build the images and push them. Latest pushed separately after the loop to avoid making each release :latest while running this script.
-  export VERSION=${VERSION}
-  ./build-images.sh
+  VERSION=${VERSION} ./build-images.sh
```

Comment:
No, the fragility comes from the `./`. If my current working directory doesn't match where I run this script from, I'll get an error:

`/tmp/test.sh: line 1: ./build-images.sh: No such file or directory`

Do this instead to avoid that issue:

```
D=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
VERSION=${VERSION} $D/build-images.sh
```